WcaOnRails/spec/factories/team_members.rb

```diff
@@ -1,8 +1,90 @@
 FactoryGirl.define do
   factory :team_member do
-    team_id 1
-    user_id 1
-    start_date "2016-02-18"
+    team
+    user
+    start_date 1.month.ago
     end_date nil
+    after(:build) do |team_member|
+      committee_position = FactoryGirl.create(:committee_position, committee_id: team_member.team.committee_id)
+      team_member.committee_position_id = committee_position.id
```

Comment:
Yeah, I've just checked and such an assignment just sets the attribute but when you reload an object the value disappears. The best solution is to do
```ruby
team_member.committee_position = committee_position
```
as it immediately produces SQL query that updates the database.